dashboard/config/application.rb

```diff
@@ -100,8 +100,6 @@ class Application < Rails::Application
       jquery.handsontable.full.css
       jquery.handsontable.full.js
     )
-    config.react.variant = :development
```

Comment:
I think that what we ultimately want is for staging to be using unminified code in dev mode, but for test/prod to be using minified code in prod mode. I think to accomplish that, we would need to start producing/using two builds (rather than letting the assets pipeline take care of that for us)